content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -1,22 +1,19 @@
 ---
 reviewers:
 - jessfraz
-title: Inject Information into Pods Using a PodPreset
+title: Inject Information into Pods Using a Pod Preset
 content_template: templates/task
 weight: 60
 ---
 
 {{% capture overview %}}
 
-You can use a `PodPreset` object to inject information like secrets, volume
-mounts, and environment variables etc into pods at creation time.
-This task shows some examples on using the `PodPreset` resource.
+This page shows how to use Pod Presets to inject information like secrets, volume mounts, and environment variables into Pods at creation time.
```

Comment:
I prefer PodPreset (if it means the object) written without a space, the same way it appears in a manifest.
With a space, I'd write it as “a Pod preset”.

Reply:
(you can use both of the ways I'm suggesting, but “Pod Preset” isn't right).

Reply:
@tom1299 , I agree with @sftim .
I find the term `Pod preset` to be confusing (https://www.ahdictionary.com/word/search.html?q=preset) especially when the resource is called PodPreset.

Also, should these changes appear on the `dev-1.18` branch or `master`?

Reply:
> I prefer PodPreset (if it means the object) written without a space, the same way it appears in a manifest.
> With a space, I'd write it as “a Pod preset”.

That does indeed make sense. I altered the document following that rationale.

Reply:
> @tom1299 , I agree with @sftim .
> I find the term `Pod preset` to be confusing (https://www.ahdictionary.com/word/search.html?q=preset) especially when the resource is called PodPreset.
> 
> Also, should these changes appear on the `dev-1.18` branch or `master`?

I tried to make the wording clearer in my last commits. Could you please take a look at the cahnges I pusehd ?
Regarding the branch; I was asked to base all of my last pull requests on the dev branch, so I did the same for this pull request. But it's no problem to target the master branch.

Reply:
@kbhawkey @sftim I would ask you for some advice on the branch issues since I do not know which would be correct.